python_client/src/decision.py

```diff
@@ -10,7 +10,7 @@
 class ScanDecision:
     """
     Attributes:
-      id, name, decision, creator, creation_time, note,
+      id, decision, creator, created, note,
```

Comment:
I would've left it at `creation_time`, which makes more sense to me, but the field in the DB is called `created` so that is what the API calls the field. I figured we should prioritize congruence with the API responses to avoid confusion.